core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/script/IScriptService.java

```diff
@@ -26,4 +27,6 @@
 
     public void deleteByNameAndVersion(String name, long version);
 
+    public void restoreByNameAndVersion(ScriptKey scriptKey);
```

Comment:
remove the file 'core/java/iesi-rest-without-microservices/null'

Reply:
Removed